litestar/__main__.py

```diff
@@ -1,10 +1,10 @@
-from litestar.cli.main import litestar_group  # pragma: no cover
+from litestar.cli.main import litestar_group
 
 
-def run_cli() -> None:  # pragma: no cover
+def run_cli() -> None:
     """Application Entrypoint."""
     litestar_group()
 
 
-if __name__ == "__main__":  # pragma: no cover
+if __name__ == "__main__":
```

Comment:
I thought these should not be required given the plugin adds this to "omit" ([ref](https://github.com/asottile/covdefaults/blob/6e03d07652445d94683f97d14953c81559bd0ba3/covdefaults.py#L84)):

```py
    ('run:omit', ['*/__main__.py', '*/setup.py']),
```